schema.sql

```diff
@@ -40,4 +40,62 @@ ALTER TABLE animals ADD owner_id INT references owners(id);
 
 --Remove column called species from animals table.
 ALTER TABLE animals DROP COLUMN species;
-     
\ No newline at end of file
+
+--Create a table named vets
+CREATE TABLE vets (
+    id serial PRIMARY KEY NOT NULL,
+    name VARCHAR(50) NOT NULL,
+    age INT,
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations(
+    vets_id INT REFERENCES vets (id),
+    species_id INT REFERENCES species (id)
+    );
+
+CREATE TABLE visits(
+    vets_id INT REFERENCES vets(id),
+    animal_id INT REFERENCES animals(id),
+    visit_date DATE
+    );
+
+--Insert the following data for vets
+INSERT INTO vets(name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+--Insert the following data for specialization
+INSERT INTO specializations (vets_id,species_id) 
+VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'),(SELECT id from species WHERE name = 'Pokemon'));
+INSERT INTO specializations (vets_id,species_id) 
+VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'),(SELECT id from species WHERE name = 'Digimon'));
+INSERT INTO specializations (vets_id,species_id)
+VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'),(SELECT id from species WHERE name = 'Pokemon'));
+INSERT INTO specializations (vets_id,species_id)
+VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'),(SELECT id from species WHERE name = 'Digimon')); 
+
+--Insert the following data for visits.
+INSERT INTO visits(animal_id, vets_id, visit_date)
```

Comment:
- [ ] the queries that insert data they should be in the `data.sql` file so it would be better if you move them there